pandas/core/arrays/categorical.py

```diff
@@ -647,8 +647,8 @@ def from_codes(cls, codes, categories=None, ordered=None, dtype=None):
             Whether or not this categorical is treated as an ordered
             categorical. If not given here or in `dtype`, the resulting
             categorical will be unordered.
-        dtype : CategoricalDtype or the string "category", optional
-            If :class:`CategoricalDtype`, cannot be used together with
+        dtype : CategoricalDtype or "category", optional
+            If :class:`Categoarray-like, intricalDtype`, cannot be used together with
```

Comment:
whoops!

Reply:
thanks for catching that :)
Are you going Man hackathon in November @simonjayhawkins ?

Reply:
yep.

Reply:
see you there :)